src/redux-ducks/library.js

```diff
@@ -54,7 +64,13 @@ const defaultState: State = {
   mangaIds: [], // array of mangaIds that point at data loaded in mangaInfos reducer
   reloadLibrary: true, // Library should be loaded once on first visit
   unread: {}, // { mangaId: int }
+  downloaded: {},
+  totalChaptersSortIndexes: {},
+  lastReadSortIndexes: {},
   reloadUnread: true, // should refresh unread for library if something new is added
+  reloadDownloaded: true,
```

Comment:
I will be splitting up the `library` endpoint to make these individually reloadable in the future.